src/main/java/core/basesyntax/FileWork.java

```diff
@@ -8,7 +15,30 @@
  * Результат: web wide width world</p>
  */
 public class FileWork {
+    public static final String FIRST_LETTER = "w";
+
     public String[] readFromFile(String fileName) {
-        return null;
+        List<String> sentences;
+        try {
+            sentences = Files.readAllLines(Paths.get(fileName));
+        } catch (IOException e) {
+            e.printStackTrace();
+            return null;
+        }
+        List<String> resultList = new ArrayList<>();
+        for (String sentence : sentences) {
+            String[] splittedSentence = sentence.toLowerCase().split(" ");
+            for (String word : splittedSentence) {
+                if (word.startsWith(FIRST_LETTER)) {
+                    resultList.add(word);
+                }
+            }
+        }
+        String[] solution = new String[resultList.size()];
```

Comment:
is there any method of List, that converts it to Array?

Reply:
Done as advised.